frontend/src/App.tsx

```diff
@@ -9,27 +9,47 @@ import Navbar from './components/Navbar'
 import SubmitTag from './components/SubmitTag'
 import RecoverAccount from './pages/RecoverAccount'
 import DiscoverFrame from './components/DiscoverFrame'
+import RequireAuth from './components/helper/RequireAuth'
 import { BrowserRouter as Router, Route, Routes} from 'react-router-dom'
 
 
 function App() {
   return (
     <Router>
-      <div className="App">
         <Navbar />
-        <div className="content">
+        <div className="mt-20 w-full h-screen bg-black">
```

Comment:
I think this tailwind styling is for background bg (black color) ?
![Screenshot_20230510_124656](https://github.com/State-Channel-4/Channel4-pocv2/assets/7667514/f5155d4d-7a99-4904-b03e-d160d63ac31e)